src/default_parameters.jl

```diff
@@ -59,23 +59,32 @@
     RKs::Int=3                          # Number of stages for SSPRK2
     RKn::Int=5                          # n^2 = s = Number of stages  for SSPRK3
     cfl::Real=0.9                       # CFL number (1.0 recommended for RK4, 0.6 for RK3)
-    Ndays::Real=200.0                   # number of days to integrate for
+    Ndays::Real=5                       # number of days to integrate for
     nstep_diff::Int=1                   # diffusive part every nstep_diff time steps.
     nstep_advcor::Int=0                 # advection and coriolis update every nstep_advcor time steps.
                                         # 0 means it is included in every RK4 substep
     compensated::Bool=false             # Compensated summation in the time integration?
 
     # BOUNDARY CONDITION OPTIONS
-    bc::String="periodic"               # "periodic" or anything else for nonperiodic
-    α::Real=2.                          # lateral boundary condition parameter
+    bc::String="nonperiodic"            # "periodic" or anything else for nonperiodic
+    α::Real=0.                          # lateral boundary condition parameter
                                         # 0 free-slip, 0<α<2 partial-slip, 2 no-slip
 
+    # PARAMETERS FOR ADJOINT METHOD
+    data_steps::StepRange{Int,Int} = 0:1:0      # Timesteps where data exists
+    data::Array{Float32, 1} = [0.]              # model data
+    J::Float64 = 0.                             # Placeholder for cost function evaluation
+    j::Int = 1                                  # For keeping track of the entry in data
+
+    # CHECKPOINTING VARIABLES
+    i::Int = 0                                  # Placeholder for current timestep, needed for Checkpointing.jl
+
     # MOMENTUM ADVECTION OPTIONS
-    adv_scheme::String="ArakawaHsu"     # "Sadourny" or "ArakawaHsu"
+    adv_scheme::String="Sadourny"       # "Sadourny" or "ArakawaHsu"
```

Comment:
Any reason why this is the new default?

Reply:
I think there might have been the thought a while back that Sadourny would be less computationally expensive? But otherwise I don't think this choice matters any longer